stylix/ordering.nix

```diff
@@ -0,0 +1,13 @@
+{ lib, ... }:
+{
+  options.stylix.listTargetIndex = lib.mkOption {
```

Comment:
Maybe this should be a font-specific option? `stylix.fonts.targetIndex` ?

I just wonder if Stylix ends up using `mkOrder` for other things, those things may end up needing different orders and/or end up needing to be configured separately?

In my initial proposal, we had a set of target indexes, which would contain named attributes for different scenarios:

E.g:
```nix
indexes = {
  before = 600;
  # ...
  font = indexes.before;
};
```

But we could also have options that are local to whatever part of stylix needs an index: `stylix.fonts.index = 600`

Reply:
All valid considerations, but this will be a private option with a single intended use at this time. So I don't think any more design is necessary. Can be easily and freely changed in the future.

Reply:
Yes, I would also prefer this to be a font-specific option. However, since the fallback implementation is still very early, it is hard to predict the extent of this order. So I am also fine keeping it as-is for the time being.